lib/rubocop/cop/lint/format_parameter_mismatch.rb

```diff
@@ -22,16 +22,22 @@ class FormatParameterMismatch < Cop
         # http://rubular.com/r/CvpbxkcTzy
         MSG = "Number of arguments (%<arg_num>i) to `%<method>s` doesn't " \
               'match the number of fields (%<field_num>i).'
+
+        FLAG = /[\s#+-0\*]/.freeze
```

Comment:
Might be a good idea to add some comments explaining what each of those are supposed to match, as it's getting a bit convoluted IMO.

Reply:
Good idea. I am adding a few pointers to the documentation I was using. I am going to squash this later.